src/odemis/acq/fastem.py

```diff
@@ -281,10 +281,12 @@ def __init__(self, scanner, multibeam, descanner, detector, stage, scan_stage, c
         # save the initial multibeam resolution, because the resolution will get updated if save_full_cells is True
         self._old_res = self._multibeam.resolution.value
 
-        path = fastem_util.create_image_dir("beam-shift-correction")
-        # The path should be [image-dir]/beam-shift-correction/[timestamp]/[roa-name]
-        self.path = os.path.join(path, self._roa.name.value)
-        os.makedirs(self.path, exist_ok=True)
+        beam_shift_path = fastem_util.create_image_dir("beam-shift-correction")
+        # The path should be [image-dir]/beam-shift-correction/[timestamp]/[project-name]/[roa-name]_[slice-idx]
+        self.beam_shift_path = os.path.join(beam_shift_path,
+                                            path,  # project-name
```

Comment:
Some of the `odemis/acq/test/fastem_test.py` test cases fail because `path=None` :(